### ISSUE TITLE
Ignore messages from bots

### ISSUE THREAD

**Issue by fdkchx** [Vencord/plugin-requests#220]
### Plugin Description

Create a plugin that will allow you to ignore or completely hide messages and chats with bots
For example, when the server bot writes to me in private messages, I will not see this message, as well as the notification itself

As an alternative, you can set up automatic search and blocking of all bots from the server on your account

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by fily-gif** [comment]
This is a great suggestion! Please never ask for selfbotting solutions.

**Comment by Sqaaakoi** [comment]
> This is a great suggestion! Please never ask for a selfbotting solutions.

????? This has nothing to do with selfbotting.

**Comment by nin0-dev** [comment]
> > This is a great suggestion! Please never ask for a selfbotting solutions.
> 
> ????? This has nothing to do with selfbotting.

they were likely talking about this
> automatic search and blocking of all bots from the server on your account

former option could be implemented, latter one not since it may be selfbotting

**Comment by EdVraz** [comment]
Automatic blocking of accounts is definitely selfbotting

**Comment by Sqaaakoi** [comment]
> > > This is a great suggestion! Please never ask for a selfbotting solutions.
> > 
> > 
> > ????? This has nothing to do with selfbotting.
> 
> they were likely talking about this
> 
> > automatic search and blocking of all bots from the server on your account
> 
> former option could be implemented, latter one not since it may be selfbotting

yeah I didnt entirely catch/consider that when I said that, you are right that auto-blocking bots is definitely selfbotting

**Comment by Andrew6rant** [comment]
Not a plugin, but you can inject this CSS to hide bot messages:

```css
[class^="messageListItem_"]:has([class^="botText_"]) {
  display:none;
}
```

It obviously won't hide notifications or perform any type of action on behalf of your account though
